bpdm-pool/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/entity/IdentifierTypeDb.kt

```diff
@@ -49,7 +50,20 @@ class IdentifierTypeDb(
     val transliteratedName: String? = null,
 
     @Column(name = "transliterated_abbreviation")
-    val transliteratedAbbreviation: String? = null
+    val transliteratedAbbreviation: String? = null,
+
+    @Column(name = "format")
+    val format: String? = null,
+
+    @ElementCollection(fetch = FetchType.EAGER)
+    @Enumerated(EnumType.STRING)
+    @CollectionTable(
+        name = "identifier_type_categories",
+        joinColumns = [JoinColumn(name = "identifier_type_id", foreignKey = ForeignKey(name = "fk_identifier_categories_types"))],
+        uniqueConstraints = [UniqueConstraint(name = "uc_identifier_categories", columnNames = ["identifier_type_id", "category"])],
+    )
+    @Column("category")
```

Comment:
```suggestion
    @Column(name = "category")
```
As only named arguments are available for Java annotations